test/cli/delegated_routing_v1_http_proxy_test.go

```diff
@@ -0,0 +1,144 @@
+package cli
+
+import (
+	"testing"
+
+	"github.com/ipfs/boxo/ipns"
+	"github.com/ipfs/kubo/config"
+	"github.com/ipfs/kubo/test/cli/harness"
+	"github.com/ipfs/kubo/test/cli/testutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoutingV1Proxy(t *testing.T) {
```

Comment:
@hacdias  Can we also disable mDNS on both?

Reply:
Done